course_catalog/etl/loaders.py

```diff
@@ -203,10 +204,11 @@ def load_video(video_data):
     load_topics(video, topics_data)
     load_offered_bys(video, offered_bys_data)
 
-    if not created and not video.published:
-        search_task_helpers.delete_video(video)
-    elif video.published:
-        search_task_helpers.upsert_video(video)
+    if features.is_enabled(features.VIDEO_INDEXING):
```

Comment:
Should the youtube task also run only if this is enabled?

Reply:
My thought was that we wanted to pull the data into the database, but not have them show up on the search page (so we could validate the data), but I think this PR has sat long enough we may not need this at all, let me check on that.

Reply:
Removed this flag after verifying with peter it's not needed